config/bootstrap.php

```diff
@@ -43,27 +42,6 @@
     ]);
 }
 
-Router::plugin('DebugKit', function ($routes) {
```

Comment:
Here could we check to see if the plugin was loaded with routes, and conditionally require_once the routes file if they forgot?